packages/scaffolding-cli/src/index.ts

```diff
@@ -47,7 +47,7 @@ async function cliCommand(argv: CliOptions): Promise<ExitMessage>{
 const runOptions = () => {
     yargs
         .scriptName("@amidostacks/scaffolding-cli")
-        .options("c", {
+        .options("config", {
             alias: ["c", "conf"],
```

Comment:
If this was failing, then the tests need to be updated to ensure that we capture this in future? @dnitsch

Reply:
they were failing for me :D that's why I changed it - see the change in prompts.test.ts